spec/features/list_spaces_spec.rb

```diff
@@ -0,0 +1,17 @@
+### User Story 2: Space Renter: Listing Spaces Available for Rent
+
+# As a space renter,
+# So that I can see available spaces to rent,
+# I'd like to search for available spaces by entering from and to dates.
+#
+# *Space Renter can only see spaces that the space owner has made available.*
+# *Space Renter can still see spaces that have a confirmed booking request within the entered dates.*
+feature 'list spaces' do
+  scenario 'list all spaces available' do
+    visit ('/spaces')
+    expect(page).to have_content 'All Spaces'
```

Comment:
We were looking for 'All Spaces' in the @spaces array in app.rb, but realised this was actually the title of the list in the view.  Not sure if there is a way we can distinguish this, or whether this will happen naturally when we add in the database link?